<launch>
<!-- Launches the lidar nodes for testing in STDR -->
<node name="lidar_interpreter" pkg="lidar" type="lidar_interpreter" args="robot0/laser_0" respawn="true"></node>	
<node name="lidar_alarm" pkg="lidar" type="lidar_alarm" respawn="true"></node>	
</launch>